node-src/ui/tasks/storybookInfo.ts

```diff
@@ -1,12 +1,19 @@
+import { isE2EBuild } from '../../lib/e2eUtils';
 import { Context } from '../../types';
+import { buildType } from './utils';
 
 const capitalize = (string: string) =>
   string
     .split('-')
     .map((str) => str.charAt(0).toUpperCase() + str.slice(1))
     .join(' ');
 
-const infoMessage = ({ addons, version, viewLayer, builder }: Context['storybook']) => {
+const infoMessage = (ctx: Context) => {
+  if (isE2EBuild(ctx.options)) {
+    return ctx.options.playwright ? 'Playwright for E2E' : 'Cypress for E2E';
```

Comment:
Should we maybe do a switch case on these?  Just thinking, if we ever add another E2E type we would catch it with exhaustive case checks.  I guess that would require some type changes too, so maybe not worth the squeeze.

Reply:
I thought about that but I wasn't sure what the `default` should be. `[unknown]` or something?

Reply:
I'm good with `E2E` since the E2E checks are a bit all over the place. I'd hate to throw here if you forget to update the UI message after changing `isE2EBuild()`.

Reply:
@jmhobbs Moved it around a bit. Let me know what you think!

Reply:
Dig it, thank you!